float64 score
int64 xmin
int64 ymin
int64 xmax
int64 ymax
geometry_msgs/Vector3 topleft
geometry_msgs/Vector3 bottomright
int16 id
string Class